lib/policy_machine_storage_adapters/template.rb

```diff
@@ -76,6 +76,15 @@ def assign(src, dst)
     end
 
     ##
+    # Assign src to dst. The two policy elements must be persisted policy
+    # elements in different policy machines. Returns true if the assignment
+    # occurred and false otherwise.
```

Comment:
This is documentation; would you mind clarifying the intent of these methods here? Something like "This is used for logical relationships outside of the policy machine formalism, such as the relationship between a class of operable and a specific instance of it."